gradle/publish.gradle

```diff
@@ -129,14 +164,19 @@ task publishGroovydoc {
 task publishRelease {
     description = "Publish release for plugin version: $tag"
     group = 'release'
-    if (project.hasProperty('dryRun') && project['dryRun'] == 'false') {
-        dependsOn prepareRelease, publishArtifact, publishGroovydoc, publishPlugins
-        doLast {
-            grgit.push {
-                tags = true
-            }
-        }
+    boolean isDryRun = buildProperties.cli['dryRun'].or(true).boolean
```

Comment:
this is used in 2 places already. could you extract as var in the outmost scope?

Reply:
What exactly do you have in mind?
We consume this value the first time within the configuration of the `buildProperties` extension itself.
We could define it within the `ext` block with a default value and override it it within the `buildPropeties.bintray` configuration. But this way we would need to rely on the order when reading the value a second time.

As alternative we could extract a method `isDryRun()`.

Reply:
extracting a method in this gradle file sounds good too (no need to expose it outside using any `ext`)